src/app/rpmostree-pkg-builtins.c

```diff
@@ -31,12 +31,14 @@ static char *opt_osname;
 static gboolean opt_reboot;
 static gboolean opt_dry_run;
 static gboolean opt_no_scripts;
+static char **opt_ignore_script;
 
 static GOptionEntry option_entries[] = {
   { "os", 0, 0, G_OPTION_ARG_STRING, &opt_osname, "Operate on provided OSNAME", "OSNAME" },
   { "reboot", 'r', 0, G_OPTION_ARG_NONE, &opt_reboot, "Initiate a reboot after upgrade is prepared", NULL },
   { "dry-run", 'n', 0, G_OPTION_ARG_NONE, &opt_dry_run, "Exit after printing the transaction", NULL },
   { "noscripts", 0, 0, G_OPTION_ARG_NONE, &opt_no_scripts, "Do not run scripts", NULL },
+  { "ignore-script", 0, 0, G_OPTION_ARG_STRING_ARRAY, &opt_ignore_script, "Ignore a script for given PACKAGE", "PACKAGE" },
```

Comment:
I'm thinking it would be nicer to just merge those two options since there's a big overlap. E.g. have an `--ignore-scripts` with an optional arg. Without the arg means the same as current `--noscripts`, and the optional arg can be of the form `pkg1,pkg2,pkg3` (or something else easily parseable).